templates/fluent-resource/en-US/security.ftl

```diff
@@ -0,0 +1,112 @@
+
+## Security coordinator email addreses and links to their public keys
+security-public-key = public key
+-security-coordinator-email-anchor =
+        <a href="mailto:steve@steveklabnik.com">Steve Klabnik</a>
+-security-coordinator-public-key-anchor =
+        <a href="https://pgp.mit.edu/pks/lookup?op=vindex&amp;search=0xDAE717EFE9424541">{ security-public-key }</a>
+-backup-security-contact-email-anchor =
+        <a href="mailto:alex@alexcrichton.com">Alex Crichton</a>
+-backup-security-contact-public-key-anchor =
+        <a href="https://pgp.mit.edu/pks/lookup?op=vindex&amp;search=0x5D54B6F551FF5E33">{ security-public-key }</a>
+
+security-disclosure-heading = Disclosure policy
+security-disclosure-p1 =
+        The Rust project has a 5 step disclosure process.
+security-disclosure-li1 =
+        The security report is received and is assigned a primary handler. This
+        person will coordinate the fix and release process.
+security-disclosure-li2 =
+        The problem is confirmed and a list of all affected versions is
+        determined.
+security-disclosure-li3 =
+        Code is audited to find any potential similar problems.
+security-disclosure-li4 =
+        Fixes are prepared for all releases which are still under maintenance.
+        These fixes are not committed to the public repository but rather held
+        locally pending the announcement.
+security-disclosure-li5 =
+        On the embargo date, the <a href="https://groups.google.com/forum/#!forum/rustlang-security-announcements">
+        Rust security mailing list</a> is sent a copy of the announcement. The
+        changes are pushed to the public repository and new builds are deployed
+        to rust-lang.org.  Within 6 hours of the mailing list being notified, a
+        copy of the advisory will be published on the Rust blog.
+security-disclosure-p2 =
+        This process can take some time, especially when coordination is
+        required with maintainers of other projects. Every effort will be made
+        to handle the bug in as timely a manner as possible, however it’s
+        important that we follow the release process above to ensure that the
+        disclosure is handled in a consistent manner.
+
+security-receiving-heading = Receiving security updates
+security-receiving-p1 =
+        The best way to receive all the security announcements is to subscribe
+        to the { -rust-security-announcements-mailing-list-anchor }
+        (alternatively by sending an email to
+        { -rustlang-security-announcements-email-anchor }). The mailing list is
+        very low traffic, and it receives the public notifications the moment
+        the embargo is lifted.
+security-receiving-p2 = 
+        We will announce vulnerabilities 72 hours before the embargo is lifted
+        to { -distros-openwall-email-anchor }, so that Linux distributions can
+        update their packages.
+
+## Security mailing list links and emails
```

Comment:
Move these to core.ftl.

(I'm going to move the `rust` key into `common.ftl` and then move `core.ftl` outside of the locale folders)

Reply:
Can do. The text "Rust security announcements mailing list" needs to be exposed to translators, so I'm thinking I'll leave the `security-rust-security-announcements-mailing-list` variable (line 103) in `security.ftl`.

Reply:
Sounds good, the email address should be in core